code/game/objects/items/weapons/clown_items.dm

```diff
@@ -41,24 +42,41 @@
 
 /obj/item/weapon/soap/afterattack(atom/target, mob/user as mob, proximity)
 	if(!proximity) return
-	//I couldn't feasibly  fix the overlay bugs caused by cleaning items we are wearing.
-	//So this is a workaround. This also makes more sense from an IC standpoint. ~Carn
-	if(user.client && (target in user.client.screen))
-		user << SPAN_NOTICE("You need to take that [target.name] off before cleaning it.")
+
 	else if(istype(target,/obj/effect/decal/cleanable))
-		user << SPAN_NOTICE("You scrub \the [target.name] out.")
+		user << "<span class='notice'>You scrub \the [target.name] out.</span>"
```

Comment:
As a general interest question.

Why would you need to change it out though?

Reply:
Sorry, there's a lot of these. It wasn't deliberate, simply that the SPAN_NOTICE and SPAN_WARNING procs didn't exist on aurora.

For reference, i deleted the entire robot_modules.dm file, and replaced it with the aurora version (which i personally worked on). I did that with a couple of smaller sections of various files, like this one about soap